docs/HowTo/Use/adding_nodes.md

```diff
@@ -68,23 +68,29 @@ differently internally.
     The new node is now up and running, and will start syncing the blockchain from existing peers. Once this has
     completed, it can send new transactions just as any other peer.
 
+!!! important
+
+    For a Raft network to work, 51% of the peers must be up and running.
+    We recommend having an odd number of at least 3 peers in a network.
+
 ### IBFT/Clique
 
 Adding nodes to an IBFT/Clique network is a bit simpler, as it only needs to configure itself rather then be
 pre-allocated on the network (permissioning aside).
 
-1. Initialise the new node with the network's genesis configuration.
+1. Initialize the new node with the network's genesis configuration.
 
-    Initialising the new node is exactly the same an the original nodes:
+    Initializing the new node is exactly the same as the original nodes:
 
     ```bash
      geth --datadir qdata/dd7 init genesis.json
     ```
 
     !!! note
-       Where you obtain this from will be dependent on the network. You may get it from an existing peer, or a network operator, or elsewhere entirely.
 
-1. If you are using permissioning or discovery for Ethereum peer-to-peer, please refer [here](#extra-options).
+        Where you obtain this from will be dependent on the network. You may get it from an existing peer, or a network operator, or elsewhere entirely.
```

Comment:
Could you replace "this" by the exact word so that it's clear what you obtain? I guess it's about he genesis file but not sure.